Modix.Services/Messages/MessageService.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Threading.Tasks;
+using Discord;
+using Discord.WebSocket;
+using Modix.Data.Repositories;
+using Modix.Services.Core;
+
+namespace Modix.Services.Messages
+{
+    /// <summary>
+    /// Describes a service that performs functions related to Discord messages.
+    /// </summary>
+    public interface IMessageService
+    {
+        /// <summary>
+        /// Asynchronously finds a message in a guild where the channel is not known.
+        /// </summary>
+        /// <param name="guildId">The unique Discord snowflake ID of the guild in which the message belongs.</param>
+        /// <param name="messageId">The unique Discord snowflake ID of the message to find.</param>
+        /// <returns>
+        /// A <see cref="Task"/> that will complete when the operation completes,
+        /// containing the message, if found, or <see langword="null"/> if not.
+        /// </returns>
+        Task<IMessage> FindMessageInUnknownChannelAsync(ulong guildId, ulong messageId);
```

Comment:
I think `FindMessageAsync()` works well enough here. Especially if we make it return `IGuildMessage`. Unless that would be problematic regarding its use in MessageTypeReader?

Reply:
That's fair. I'll look into returning `IGuildMessage` as well